src/components/phone/ContactCard.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div>
+    <div class="flex flex-row justify-between">
+      <div class="flex-col m-3">
+        <img :src="profileSrc" alt="PFP" class="rounded-full" />
+      </div>
+      <div class="flex-col justify-between m-3">
+        <div class="flex flex-row justify-between">
+          <div class="flex flex-col justify-between">
+            <base-text variant="h2" class="mr-auto">{{ name }}</base-text>
+          </div>
+          <div class="flex-col">
+            <base-button
+              icon-size="md"
+              size="md"
+              ccu-icon="edit"
+              class="hover:bg-crisiscleanup-grey-100 ml-auto"
+            />
+          </div>
+        </div>
+        <base-text variant="h2" weight="300" class="justify-between">{{
+          mobile
+        }}</base-text>
+        <div id="moreinfobtn" class="more-infobtn">
+          <base-button
+            class="text-crisiscleanup-dark-blue align-right"
+            :action="() => (toggleOpen = !toggleOpen)"
+            >{{ !toggleOpen ? 'more' : 'less' }} info
+          </base-button>
+        </div>
+      </div>
+    </div>
+    <!--- info card --->
+    <div class="info-card">
+      <more-info v-if="toggleOpen" />
+    </div>
+    <!--- Buttons --->
+    <div class="flex flex-row justify-around items-center m-3">
+      <div class="flex-col">
+        <base-button
+          v-if="completedTraining"
```

Comment:
Hate to do this to you... but this component no longer exists :man_shrugging: 

As of 7968b72 I made a lot of changes to it and moved it to `src/components/phone/AgentCard.vue`